Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "new_mean = 400\n",
+    "bg_mean = 100\n",
+    "new_sample = stats.poisson.rvs(new_mean, size=10000)\n",
+    "al = np.array([])\n",
+    "for i in new_sample:\n",
+    "    al = np.append(al, i - bg_mean)\n",
+    "    \n",
+    "plt.hist([new_sample, al], bins = 250, color=['lightblue', 'orange'], label=['Total observed counts', 'Injected signal counts'])\n",
+    "plt.title('Signal and Total Counts')\n",
+    "plt.xlabel('Measurements')\n",
+    "plt.ylabel('Counts')\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can confirm again that these distributions look like previous distributions. The injected signal distribution is independent of the background, it is made from the signal alone and without contribution from the background."
```

Comment:
Problem 3D-3E:
- Math: Normalizing first makes sense and output looks good I think. Is the r = 1 value in your math the sigma value?
- Explanation looks good. End of explanation is particularly good discussing symmetry. I think this portion is good as is.